bin/qtile-session

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+while true; do
+    qtile >/dev/null 2>&1
```

Comment:
I wonder if we shouldn't leave the output, since that's what ~/.xsession-errors shows, and we often ask people to look there when debugging stuff?

That said, can we just drop this and added it to the sample .xinitrc you put below?

Reply:
I think we can